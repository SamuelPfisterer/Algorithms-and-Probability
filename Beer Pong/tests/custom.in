5
2 3 5 1
0.3 1.0
2 2 5 1
0.3 1.0
2 2 3 2
0.3 0.9
2 2 5 2
0.3 0.9
2 2 3 3
0.3 0.9
