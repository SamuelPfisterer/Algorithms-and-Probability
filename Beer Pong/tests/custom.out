0.1875
0.2142857
0.1391753
0.0927835
0.3303
